src/utils/css/css.js

```diff
@@ -1,11 +1,11 @@
 export default {
-  clearfix: "clearfix",
-  input: "common-input",
-  hidden: "hidden",
-  unselectable: "unselectable",
-  textTag: "text--tag",
-  textBold: "text--bold",
-  textItalic: "text--italic",
-  textSecondary: "text--secondary",
-  textInactive: "text--inactive"
+  clearfix: 'clearfix',
+  input: 'common-input',
+  hidden: 'hidden',
+  unselectable: 'unselectable',
+  textTag: 'text--tag',
+  textBold: 'text--bold',
+  textItalic: 'text--italic',
+  textSecondary: 'text--secondary',
+  textInactive: 'text--inactive'
```

Comment:
I'm only doing linting in order not to confuse matters.

I agree, we should do this sort of thing but think it obfuscates the PR related changes.